README.md

```diff
@@ -92,5 +92,16 @@ fastify.route({
 })
 ```
 
+## Plugins
+You can also extend the functionalities of the ajv instance embedded in this validator by adding new ajv plugins.
```

Comment:
Please mention it is not using the latest `ajv` version. The users should install the corresponding version of plugin.
https://github.com/fastify/fastify-response-validation/df4e7c1aaa395e59c2d74d2299a6e610a223fb51/master/package.json#L7

Reply:
Well. nowadays we use ajv 8. So maybe this PR can be reviewed again

Reply:
Yes, this comment is resolved. But still lacks of unit test.